source/_integrations/lock.mqtt.markdown

```diff
@@ -145,6 +145,14 @@ json_attributes_topic:
   description: The MQTT topic subscribed to receive a JSON dictionary payload and then set as sensor attributes. Usage example can be found in [MQTT sensor](/integrations/sensor.mqtt/#json-attributes-topic-configuration) documentation.
   required: false
   type: string
+motor_state_topic:
+  description: The MQTT topic subscribed to receive motor state updates. It accepts states the `state_jammed`, `state_ok`.
```

Comment:
```suggestion
  description: The MQTT topic subscribed to receive motor state updates. It accepts the states `state_jammed` and `state_ok`.
```
Only two inverted words and isn't it better to replace the comma by "and" for a list of two?

Reply:
The motor state topic is removed after discussing this with @emontnemery. So we are going to use a single state topic, like you suggested at first.